addons/packages/app-toolkit/0.2.0/bundle/config/developer-namespace.yaml

```diff
@@ -0,0 +1,48 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:yaml", "yaml")
+
+#@ if/end data.values.developer_namespace != "default":
+---
+apiVersion: v1
+kind: Namespace
```

Comment:
Does kapp correctly handle the .spec.finalizers array that namespaces have?

Reply:
**Notes from Zoom Call:**
Seems that `kapp` does, but even if it doesn't it should. Not our problem 😛